aether-kernel/aether/kernel/api/views.py

```diff
@@ -32,14 +32,13 @@
 from rest_framework.renderers import JSONRenderer
 
 from aether.sdk.multitenancy.views import MtViewSetMixin
-
-from .avro_tools import extract_jsonpaths_and_docs
-from .constants import LINKED_DATA_MAX_DEPTH
-from .entity_extractor import (
+from aether.python.entity.extractor import (
     extract_create_entities,
```

Comment:
I assume this method is only called if offline extraction is disabled?

Reply:
Yes that is correct, What I have in this PR is simply the current working kernel implementation using the new python library. No `entity extraction module` link yet. The idea is to break the large `exm` PR  ie: (pythonlib + exm + submissions validation) into smaller PRs. This [PR](https://github.com/eHealthAfrica/aether/pull/649) will be updated to contain the complete abstraction of entity extraction with this as a dependency so as to use the `pythonlib` within `exm`